### PR TITLE
Create a Recommendation File for Exeter

### PR DESCRIPTION
**Pull request by emilym-sba**
Things to Do (or something it doesn't matter):

- [x] Add commit
- [x] Review comment
- [x] Official Review
- [x] Merge

### CODE REVIEW

**Comment by luizgglopes** [comment]
This is a general comment 😄

File: emilym-sba-exeter.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- A&W
+- My Mom's House
+
+### Places to Visit
+- Kaweah Lake
+- General Sherman
+
+### Places to Avoid
+- The Orange Groves
+- Woodlake
```
Comment (luizgglopes) [inline emilym-sba-exeter.md]: 🆗

Comment (emilym-sba) [reply emilym-sba-exeter.md]: 🆒

File: emilym-sba-exeter.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Exeter, CA
+
+### Places to Eat
+- A&W
+- My Mom's House
```
Comment (luizgglopes) [inline emilym-sba-exeter.md:5]: 😆

**Review by luizgglopes — APPROVED** [review]

**Pull request merged**